src/dev-app/slider/slider-demo.html

```diff
@@ -1,13 +1,17 @@
-<h1>Color Picker</h1>
+<mat-checkbox [color]="colorModel" [(ngModel)]="discrete">Discrete</mat-checkbox>
+<br>
+<mat-checkbox [color]="colorModel" [(ngModel)]="showTickMarks">Show tick marks</mat-checkbox>
+<br>
 <mat-button-toggle-group [(ngModel)]="colorModel" aria-label="Slider color options">
   <mat-button-toggle value="primary">Primary</mat-button-toggle>
   <mat-button-toggle value="accent">Accent</mat-button-toggle>
   <mat-button-toggle value="warn">Warn</mat-button-toggle>
 </mat-button-toggle-group>
+<br>
```

Comment:
You didn't introduce this, but we might as well fix it while we're here. We shouldn't be using `br` tags for spacing.